### DISCUSSION TITLE
Connecting ReactiveBase to local ElasticSearch DB

### DISCUSSION THREAD

**Discussion by kklhuang** [appbaseio/reactivesearch#2047] [Q&A] [answered]
### Discussed in https://github.com/appbaseio/reactivesearch/discussions/2019

<div type='discussions-op-text'>

<sup>Originally posted by **kklhuang** July  7, 2022</sup>
Hi! I'm currently following the book search tutorial here: https://docs.appbase.io/docs/reactivesearch/v3/overview/reactivebase/ 
However, I'm trying to connect to my own local ElasticSearch DB on port 9200. 

I've referenced this issue discussion: https://github.com/appbaseio/reactivesearch/issues/104 

This is what I've changed: 

App.js: 
<img width="569" alt="Screen Shot 2022-07-07 at 1 39 43 PM" src="https://user-images.githubusercontent.com/56480921/177835910-3d2af227-19e8-4347-9ca8-16aee1f85d68.png">

elasticsearch.yml: 
<img width="1059" alt="Screen Shot 2022-07-07 at 1 40 11 PM" src="https://user-images.githubusercontent.com/56480921/177835980-7bbd6de8-59cf-4095-8706-3ac1cee9c088.png">

And this is my local db:
![Screen Shot 2022-07-07 at 1 41 02 PM](https://user-images.githubusercontent.com/56480921/177836141-e4a1e6ae-4838-4044-9080-7e85f0b079d6.png)

However, when I'm trying to get the data from each item to display on my localhost:3000 React page, nothing shows up. Is anyone able to help?

</div>

**Comment by siddharthlatest** [comment, marked as answer]
@kklhuang With `enableAppbase`, it's expected that the connection is with ReactiveSearch API server https://github.com/appbaseio/reactivesearch-api.

You can set `enableAppbase={false}` if you intend to directly connect to Elasticsearch.
